src/spikeinterface/postprocessing/correlograms.py

```diff
@@ -120,62 +168,56 @@ def _make_bins(sorting, window_ms, bin_ms):
     return bins, window_size, bin_size
 
 
-def compute_autocorrelogram_from_spiketrain(spike_times, window_size, bin_size):
+def _compute_num_bins(window_size, bin_size):
     """
-    Computes the auto-correlogram from a given spike train.
-
-    This implementation only works if you have numba installed, to accelerate the
-    computation time.
+    Internal function to compute number of bins, expects
+    window_size and bin_size are already divisible. These are
+    typically generated in `_make_bins()`.
 
-    Parameters
-    ----------
-    spike_times: np.ndarray
-        The ordered spike train to compute the auto-correlogram.
-    window_size: int
-        Compute the auto-correlogram between -window_size and +window_size (in sampling time).
-    bin_size: int
-        Size of a bin (in sampling time).
     Returns
     -------
-    tuple (auto_corr, bins)
-    auto_corr: np.ndarray[int64]
-        The computed auto-correlogram.
+    num_bins : int
+        The total number of bins to span the window, in samples
+    half_num_bins : int
+        Half the number of bins. The bins are an equal number
+        of bins that look forward and backwards from zero, e.g.
+        [..., -10 to -5, -5 to 0, 0 to 5, 5 to 10, ...]
+
     """
-    assert HAVE_NUMBA
-    return _compute_autocorr_numba(spike_times.astype(np.int64), window_size, bin_size)
+    num_half_bins = int(window_size // bin_size)
+    num_bins = int(2 * num_half_bins)
 
+    return num_bins, num_half_bins
 
-def compute_crosscorrelogram_from_spiketrain(spike_times1, spike_times2, window_size, bin_size):
+
+def _compute_correlograms_on_sorting(sorting, window_ms, bin_ms, method="auto"):
     """
-    Computes the cros-correlogram between two given spike trains.
+    Computes several cross-correlogram in one course from several clusters.
```

Comment:
Do we want cluster or units for our terms?

Reply:
Good Q (another one for the glossery!), I prefer `unit` I think WDYT?

Reply:
I'm firmly on unit. I think cluster is a hold out from thinking about this in mathy terms rather than neuroscience-y terms. So I think unit is clearer for most people.